osutil/flock_test.go

```diff
@@ -67,8 +67,10 @@ func (s *flockSuite) TestLockUnlockWorks(c *C) {
 
 	// Run a flock command in another process, it should succeed because it can
 	// lock the lock as we didn't do it yet.
-	cmd := exec.Command("flock", "--exclusive", "--nonblock", lock.Path(), "true")
+	// use --no-fork so that the kill command below works
+	cmd := exec.Command("flock", "--no-fork", "--exclusive", "--nonblock", lock.Path(), "true")
```

Comment:
Ubuntu 14.04 doesn't have --no-fork, I believe this is why I didn't use it.

Reply:
Uh, that is very unfortunate

Reply:
I pushed an update that works without --no-fork